platform/darwin/src/MGLMapCamera.mm

```diff
@@ -91,6 +92,7 @@ - (instancetype)initWithCenterCoordinate:(CLLocationCoordinate2D)centerCoordinat
                                    pitch:(CGFloat)pitch
                                  heading:(CLLocationDirection)heading
 {
+    MGLLogDebug(@"Initializing withCenterCoordinate: %@ altitude: %.0fm pitch: %f° heading: %f°", NSStringFromCLLocationCoordinate2D(centerCoordinate), altitude, pitch, heading);
```

Comment:
Note that this initializer is called from other initializers, so the parameters may not reflect the caller’s explicit parameters.

Reply:
Although this method may get called from the class constructors it does reflects the values these passed. Perhaps I should log the constructor methods to reflect the values that were passed to them.